### ISSUE TITLE
The box 'geerlingguy/ubuntu1604' could not be found or could not be accessed in the remote catalog.

### ISSUE THREAD

**Issue by rkoberg** [geerlingguy/drupal-vm#1010]
I see some closed issues for this, but getting the following errors. How to fix?

OSX 10.12.1
VirtualBox Version 5.1.10 r112026 (Qt5.6.2)
```
$ vagrant --version
Vagrant 1.8.7
$ ansible --version
ansible 2.3.0 (devel d61b2ed0a3) last updated 2016/11/23 07:10:32 (GMT -700)
  lib/ansible/modules/core: (detached HEAD b598611afb) last updated 2016/11/23 07:10:40 (GMT -700)
  lib/ansible/modules/extras: (detached HEAD 25292b3ebd) last updated 2016/11/23 07:10:40 (GMT -700)
  config file = 
  configured module search path = Default w/o overrides
```
And the errors:
```
[rkoberg@imac ~/Sites/drupal-vm (master)]$ vagrant up
Bringing machine 'drupalvm' up with 'virtualbox' provider...
==> drupalvm: Box 'geerlingguy/ubuntu1604' could not be found. Attempting to find and install...
    drupalvm: Box Provider: virtualbox
    drupalvm: Box Version: >= 0
The box 'geerlingguy/ubuntu1604' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Atlas, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://atlas.hashicorp.com/geerlingguy/ubuntu1604"]
Error: 
[rkoberg@imac ~/Sites/drupal-vm (master)]$ vagrant box add geerlingguy/ubuntu1604
The box 'geerlingguy/ubuntu1604' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Atlas, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://atlas.hashicorp.com/geerlingguy/ubuntu1604"]
Error: 
[rkoberg@imac ~/Sites/drupal-vm (master)]$ vagrant box add geerlingguy/ubuntu1604 https://atlas.hashicorp.com/geerlingguy/boxes/ubuntu1604/versions/1.0.4/providers/virtualbox.box
==> box: Box file was not detected as metadata. Adding it directly...
==> box: Adding box 'geerlingguy/ubuntu1604' (v0) for provider: 
    box: Downloading: https://atlas.hashicorp.com/geerlingguy/boxes/ubuntu1604/versions/1.0.4/providers/virtualbox.box
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.


[rkoberg@imac ~/Sites/drupal-vm (master)]$ vagrant init geerlingguy/ubuntu1604; vagrant up --provider virtualbox
`Vagrantfile` already exists in this directory. Remove it before
running `vagrant init`.
Bringing machine 'drupalvm' up with 'virtualbox' provider...
==> drupalvm: Box 'geerlingguy/ubuntu1604' could not be found. Attempting to find and install...
    drupalvm: Box Provider: virtualbox
    drupalvm: Box Version: >= 0
The box 'geerlingguy/ubuntu1604' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Atlas, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://atlas.hashicorp.com/geerlingguy/ubuntu1604"]
Error: 
[rkoberg@imac ~/Sites/drupal-vm (master)]$ 
```

**Comment by geerlingguy** [comment]
Please see notes in: https://github.com/geerlingguy/drupal-vm/issues/981 — either run the fix in that issue, or download Vagrant 1.8.6 for now.

**Comment by rkoberg** [comment]
Thank you for the quick response. The fix you referenced https://github.com/geerlingguy/drupal-vm/issues/981 worked for me:
```
$ sudo rm -rf /opt/vagrant/embedded/bin/curl
```
